### ISSUE TITLE
Changing the voice language but keeping the texts in a different one

### ISSUE THREAD

**Issue by lordhersiker** [Limpster/Fallout76LanguageChanger#1]
Would it be possible to modify the script so it changes only the voice audio languages so, for example, someone could play with the English dub while keeping the texts in Spanish, German or whatever?

**Comment by Limpster** [comment]
tl;dr: yes, but **not** (now) with this script, see the steps below

As of now that's not in the scope of this script, because (from my understanding) it's only possible to achieve by modifying the files e.g. renaming the audio files to the corresponding language (maybe it is possible with linking/pointing to the right files and "making the game think it's using the right files" without actually moving/renaming the audio files.
Here is a possible solution, a quick test ran fine and I had english audio with german text:
(please take it with a grain of salt since I didn't look too much into this option, the changes are experimental and I don't know if the follwing files are everything you need to change or if the game will crash in certain situations because one file is missing; don't forget to back up your files if you don't want to download/verify the files again in case something goes wrong or you want to revert the changes). Bonus: for this change you don't need my script, it's "permanent" (at least until the next update if they add new voice files which need to be renamed, see below)

What will happen:
We are renaming the english audio files, the game will load german audio, but we replaced the german audio with the english audio files.
(Example for german text with english audio, if you want to have es/fr/it/ja text, follow the steps with the corresponding language code, these are all the supported audio languages)
1. Navigate to your game folder and open the "Data" Folder (e.g. C:\XboxGames\Fallout 76\Content\Data)
2. inside this folder you will see plenty of files, we are especially interested in the "*voices_de.ba2" files ("de"=german, "es"=spanish, "fr"=french, "it"=italian, "ja"=japanese, these are all the supported audio languages)
3. make a backup of all the "*voices_de.ba2" files (=german audio), e.g. rename them to "_*voices_de.ba2"
4. make a copy of all the "*voices.ba2" files (=english audio) (you now have to sets of english audio files)
5. rename all the copies of the english audio files and give them the name of the german audio files e.g. "SeventySix - 00UpdateVoices.ba2" will become "SeventySix - 00UpdateVoices_de.ba2" (you now have 3 sets of audio files:: the german audio files backup, the english original files and the english audio files which you renamed to the german audio files)
At the time of writing this instruction you might have to backup/rename 15 files:

| Original Files                  | Renamed to                           |
|---------------------------------|--------------------------------------|
| SeventySix - 00UpdateVoices.ba2 | SeventySix - 00UpdateVoices_de.ba2   |
| SeventySix - 01UpdateVoices.ba2 | SeventySix -   01UpdateVoices_de.ba2 |
| SeventySix - 02UpdateVoices.ba2 | SeventySix -   02UpdateVoices_de.ba2 |
| SeventySix - 03UpdateVoices.ba2 | SeventySix -   03UpdateVoices_de.ba2 |
| SeventySix - 04UpdateVoices.ba2 | SeventySix -   04UpdateVoices_de.ba2 |
| SeventySix - 05UpdateVoices.ba2 | SeventySix -   05UpdateVoices_de.ba2 |
| SeventySix - 06UpdateVoices.ba2 | SeventySix -   06UpdateVoices_de.ba2 |
| SeventySix - 07UpdateVoices.ba2 | SeventySix -   07UpdateVoices_de.ba2 |
| SeventySix - 08UpdateVoices.ba2 | SeventySix -   08UpdateVoices_de.ba2 |
| SeventySix - 09UpdateVoices.ba2 | SeventySix -   09UpdateVoices_de.ba2 |
| SeventySix - 10UpdateVoices.ba2 | SeventySix -   10UpdateVoices_de.ba2 |
| SeventySix - 11UpdateVoices.ba2 | SeventySix -   11UpdateVoices_de.ba2 |
| SeventySix - 12UpdateVoices.ba2 | SeventySix -   12UpdateVoices_de.ba2 |
| SeventySix - 13UpdateVoices.ba2 | SeventySix -   13UpdateVoices_de.ba2 |
| SeventySix - Voices.ba2         | SeventySix -   Voices_de.ba2         |

And that's it, if you run the game now (without my script! just run it from the launcher/shortcut on your desktop etc.) the game should show the text in your system language (e.g like it would run without any modification) but you should be able to hear english audio.

This change is permanent, until they add more audio files or the files become replaced from the launcher. You don't need my Fallout76LanguageChanger for this.

I will add this section to the readme and see if I can find a way to create a script for this option (it's not on my priority list, but I keep it in mind). There might be a way to temporary "rename" the files with a script so we don't have to copy files/actually rename them.

**Feedback appreciated, since I don't know if this will work properly!**

**Comment by Limpster** [comment]
added "Experimental" section to the readme

**Comment by Shadesoul** [comment]
Just say thank you Limpster, I'm enjoying the game now. You are my hero :) 
And god damn Bethesda and microsoft
